src/Plugin/GraphQL/SchemaExtension/ThunderSchemaExtensionPluginBase.php

```diff
@@ -98,12 +98,12 @@ protected function resolveBaseFields(string $type) {
 
 
   /**
-   * Add fields common to all content types.
+   * Add fields common to all page types.
    *
    * @param string $type
    *   The type name.
    */
-  protected function resolveContentTypeInterfaceFields(string $type) {
+  protected function resolvePagesInterfaceFields(string $type) {
```

Comment:
Shouldnt it be resolvePageInterfaceFields (without s in Pages) like resolveContentTypeInterfaceFields?